docs/machine-learning/how-to-guides/load-data-model-builder.md

```diff
@@ -0,0 +1,108 @@
+
+## Model Builder scenarios 
+
+Model Builder helps you create models for the following machine learning scenarios:
+
+- Sentiment analysis (binary classification): Classify textual data into two categories.
+- Issue classification (multiclass classification): Classify textual data into 3 or more categories.
+- Price prediction (regression): Predict a value from numeric data.
+- Image classification (deep learning): Categorize images based on characteristics.
+- Custom scenario: Build custom scenarios from your data using regression, classification, and other tasks.
+
+This article covers classification and regression scenarios with textual or numerical data, and image classification scenarios. 
+
+## Load text or numeric data from a file  
+
+You can load text or numeric data from a file into Model Builder. It accepts comma-delimited (CSV) or tab-delimited (TSV) file formats. 
+
+1. In the data step of the Model Builder tool, select **File** from the data source dropdown.
+2. Select the button next to the **Select a file** text box, and use File Explorer to browse and select the data file.
+3. Choose a category in the **Column to Predict (Label)** dropdown.
+4. From the **Input Columns (Features)** dropdown, confirm the data columns you want to include are checked.
+
+You're done setting up your data source file for Model Builder. Select the **Train** link to move to the next step in the Model Builder tool.
+
+## Load data from a SQL Server database
+
+Model Builder supports loading data from local and remote SQL Server databases. To use data from SQL Server, you'll first download it to a file. [QUESTION: Is this true? It's confusing -- connect to the database or download??]
+
+To load data from a SQL Server database into Module Builder:
+
+1. In the data step of the Model Builder tool, select **SQL Server** from the data source dropdown.
+1. Select the button next to the **Connect to SQL Server database** text box.
+    1. In the **Choose Data** dialog, select **Microsoft SQL Server Database File**. 
+    1. Uncheck the **Always use this selection** checkbox and select **Continue**
+    1. In the **Connection Properties** dialog, select **Browse** and select the downloaded .MDF file.
+    1. Select **OK**
+1. Choose the data set name from the **Table Name** dropdown.
+1. From the **Column to Predict (Label)** dropdown, choose the data category on which you want to make a prediction.
+1. From the **Input Columns (Features)** dropdown, confirm the columns you want to include are checked. 
+
+You're done setting up your data source file for Model Builder. Select the **Train** link to move to the next step in the Model Builder tool.
+
+## Set up image data files
+
+Model Builder expects image data to be JPG files organized in folders that correspond to the categories of the classification. 
```

Comment:
```suggestion
Model Builder expects image data to be JPG or PNG files organized in folders that correspond to the categories of the classification. 
```

I need to clarify the limitations on file format, but I believe JPG and PNG are supported